openslides_backend/action/actions/meeting_user/create.py

```diff
@@ -18,27 +21,21 @@ class MeetingUserCreate(MeetingUserMixin, CreateAction):
     schema = DefaultSchema(MeetingUser()).get_create_schema(
         required_properties=["user_id", "meeting_id"],
         optional_properties=[
-            "comment",
-            "number",
-            "structure_level",
             "about_me",
```

Comment:
Why is "about_me" special?

Reply:
Because you can change it, if it is the request-user's `about_me`, see action documentation